### PR TITLE
Update Traditional Chinese

### PR DESCRIPTION
**Pull request by yichifauzi**
Update Traditional Chinese

### CODE REVIEW

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 1]
```diff
@@ -1,63 +1,86 @@
...
-	"subtitles.pf.gravel.crunch": "礫石碰撞聲",
-	"subtitles.pf.ice_crack": "冰塊碎裂聲",
-	"subtitles.pf.mud_squealk": "泥巴踩踏聲",
-	"subtitles.pf.glass_rattles": "玻璃踩踏聲",
-	"subtitles.pf.glass_croak": "玻璃彎曲低鳴聲",
-	"subtitles.pf.chain": "鎖鏈晃動聲",
-	"subtitles.pf.stone_crack": "石頭碎裂聲",
-	"subtitles.pf.wax_squeak": "蠟吱吱聲"
-}
+    "mod.presencefootsteps.name": "臨場腳步聲 Presence Footsteps",
```
Comment (Sollace) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:2]: Can you fix the indentation please? Should be 1 tab and the same for the entire file.

Comment (yichifauzi) [reply src/main/resources/assets/presencefootsteps/lang/zh_tw.json]: Okay , I fixed the indentation
thanks you

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 2]
```diff
@@ -1,63 +1,86 @@
...
+    "menu.pf.volume.passive_entities.tooltip": "改變動物腳步的音量。",
+    "menu.pf.volume.player": "用戶端玩家：%d%%",
+    "menu.pf.volume.player.tooltip": "改變自己腳步聲的音量。",
+    "menu.pf.volume.other_players": "其他玩家：%d%%",
+    "menu.pf.volume.other_players.tooltip": "在多人遊戲中改變其他玩家腳步聲的音量。",
+    "menu.pf.volume.running": "跑步增加音量：%d%%",
+    "menu.pf.volume.running.tooltip": "跑步時腳步聲音量增加多少。\n\n這個值將「加入」到總音量中，會隨著你的速度而縮放。\n\n建議先將其設為較低的值，然後再根據需求進行調整。較高的值會使效果更明顯。",
+    "menu.pf.volume.wet": "潮濕的表面：%s",
+    "menu.pf.volume.wet.tooltip": "變更在潮濕方塊上行走時產生的音量。",
+    "menu.pf.volume.foliage": "葉子: %s%%",
```
Comment (notlin4) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:36]:
```suggestion
    "menu.pf.volume.foliage": "葉子：%s%%",
```

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 3]
```diff
@@ -1,63 +1,86 @@
...
+    "menu.pf.group.footsteps": "腳步聲",
+    "menu.pf.group.debugging": "除錯",
+    "menu.pf.group.sound_packs": "音效包",
+    "pf.report.save": "檔案儲存為：%s",
+    "pf.report.error": "生成報告失敗：%s",
+    "pf.update.title": "有可用的更新",
+    "pf.update.text": "臨場腳步聲 %s",
+    "pf.update.checking": "正在檢查更新...",
+    "pf.update.updates_available": "有可用的更新！\n臨場腳步聲 %s 適用於 Minecraft %s",
+    "pf.update.up_to_date": "你使用的已是最新版",
```
Comment (notlin4) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:63]:
```suggestion
    "pf.update.up_to_date": "你使用的是最新版本",
```

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 4]
```diff
@@ -1,63 +1,86 @@
...
+    "menu.pf.multiplayer.true": "單人 + 多人遊戲",
+    "menu.pf.multiplayer.false": "單人遊戲",
+    "menu.pf.global.all": "實體：全部",
+    "menu.pf.global.players_and_hostiles": "實體：玩家 + 敵對",
+    "menu.pf.global.players_only": "實體：僅玩家",
+    "menu.pf.volume.min": "全域音量：關閉",
+    "menu.pf.disable_mod": "停用所有聲音",
+    "menu.pf.volume": "全域音量：%d%%",
+    "menu.pf.volume.tooltip": "臨場腳步聲所有聲音的全域音量。如果你想完全停用此模組，請設為 0。",
+    "menu.pf.volume.hostile_entities": "敵對生物: %d%%",
```
Comment (notlin4) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:20]:
```suggestion
    "menu.pf.volume.hostile_entities": "敵對生物：%d%%",
```

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 5]
```diff
@@ -1,63 +1,86 @@
...
+    "menu.pf.volume.hostile_entities.tooltip": "改變敵對生物腳步聲的音量。",
+    "menu.pf.volume.passive_entities": "友好生物: %d%%",
```
Comment (notlin4) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:22]:
```suggestion
    "menu.pf.volume.passive_entities": "友好生物：%d%%",
```

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 6]
```diff
@@ -1,63 +1,86 @@
...
+    "menu.pf.footwear.on": "鞋子: 開",
```
Comment (notlin4) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:23]:
```suggestion
    "menu.pf.footwear.on": "鞋子：開",
```

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 7]
```diff
@@ -1,63 +1,86 @@
...
+    "menu.pf.footwear.off": "鞋子: 關",
```
Comment (notlin4) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:24]:
```suggestion
    "menu.pf.footwear.off": "鞋子：關",
```

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 8]
```diff
@@ -1,63 +1,86 @@
...
+    "menu.pf.exclusive_mode.on": "專屬模式: 開",
```
Comment (notlin4) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:25]:
```suggestion
    "menu.pf.exclusive_mode.on": "專屬模式：開",
```

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 9]
```diff
@@ -1,63 +1,86 @@
...
+    "menu.pf.exclusive_mode.off": "專屬模式: 關",
```
Comment (notlin4) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:26]:
```suggestion
    "menu.pf.exclusive_mode.off": "專屬模式：關",
```

File: src/main/resources/assets/presencefootsteps/lang/zh_tw.json [hunk 10]
```diff
@@ -1,63 +1,86 @@
...
+    "menu.pf.stance.flying_biped.tooltip": "禽鳥：模擬兩條腿帶翅膀的走路方式",
+    "menu.pf.report.full": "完整報告",
+    "menu.pf.report.concise": "簡潔報告",
+    "menu.pf.report.acoustics": "聲學報告",
+    "menu.pf.group.volume": "音量",
+    "menu.pf.group.footsteps": "腳步聲",
+    "menu.pf.group.debugging": "除錯",
+    "menu.pf.group.sound_packs": "音效包",
+    "pf.report.save": "檔案儲存為：%s",
+    "pf.report.error": "生成報告失敗：%s",
```
Comment (notlin4) [inline src/main/resources/assets/presencefootsteps/lang/zh_tw.json:58]:
```suggestion
    "pf.report.error": "無法產生報告：%s",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Review by Sollace — CHANGES_REQUESTED** [review]
Still waiting for you to fix the indentation.

**Review by Sollace — APPROVED** [review]
Thanks

**Pull request merged**